styleguide/procedures-instructions/describing-interactions-with-ui.md

```diff
@@ -22,11 +23,11 @@ input method. Don't use input-specific verbs, such as *click* or *swipe*. Instea
+|       **Select and hold, select and hold (or right-click)**       |                                                                                                                                                                                                   Use to describe pressing and holding an element in the UI. It's OK to use *right-click* with *select and hold* when the instruction isn't specific to touch devices.                                                                           |                                                                                                                                                                                                                                                                <ul><li>To flag a message that you want to deal with later, select and hold it, and then select **Set flag**.</li></ul> <ul><li>Select and hold (or right-click) the Windows taskbar, and then select **Cascade windows**.</li></ul> <ul><li>Select and hold (or right-click) the **Start** :::image type="icon" source="media/describing-interactions-with-ui/967781121.png"::: button, and then select **Device Manager**.</li></ul>                                                                                                                             |
 |              **>**              |                                                                                                   Use a greater-than symbol (>) to separate sequential steps.<br /><br />Only use this approach when there's a clear and obvious path through the UI and the selection method is the same for each step. For example, don't mix things that require opening, selecting, and choosing.<br /><br />Don't bold the greater-than symbol. Include a space before and after the symbol.                                                                                                   |                                                                                                                                                                                                                                                                                       Select **Accounts > Other accounts > Add an account**.                                                                                                                                                                                                                                                                                        |
```

Comment:
The example blatantly disobeys the rule it shows. Can you fix that too? ("don't bold the char")